src/commcare_cloud/ansible/deploy_commcarehq.yml

```
@@ -73,22 +73,22 @@
   tags: services
 
 - name: Management Command Supervisor Config
-  hosts: commcarehq
+  hosts: commcarehq:formplayer
```

If the HQ code isn't on formplayer machines then we can't run management command from them either.

```suggestion
  hosts: commcarehq
```

---

What management commands do we need to run on a formplayer machine?

---

@millerdev none, hence why I suggested removing it.

---

I'm sorry, I misinterpreted your comment. I thought you were saying we need to keep the commcarehq source on these machines so we can run management commands on them.